docs/resources/samples/clear-table-filter-for-active-cell.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Remove table column filters
 description: Learn how to clear table column filter based on active cell location.
+ms.topic: sample
```

Comment:
consider expanding the ms.topic entry in the docfx.json so all the .md files in the samples and scenarios folders are set to sample